packages/animations/README.md

```diff
@@ -10,6 +10,89 @@ Currently, the following animated effects are available in this library:
 
 ### Material's [Open Container Transitions](https://material.io/design/motion/choreography.html#transformation)
 
-Tapping on a container  (e.g. a card or a button) will expand the container to reveal more information.
+Tapping on a container (e.g. a card or a button) will expand the container to reveal more information.
 
-*TODO(goderbauer): Add example videos of this effect.*
+1) Fade Transition
+
+- Card Example
+
+<p align="center">
```

Comment:
Having just one demo per line looks a little "lost". Maybe we should have three or for demo gifs next to each other with a short subtitle?